### PR TITLE
Material-UI 4 Migration

### PR DESCRIPTION
**Pull request by kevinjihwanlee**
This PR migrates our repository from Material UI v3 to v4 following [this guide](url). The following changes had to be made:
- Updated Typography components to use new API and removed deprecation console logs
- Updated Paper components to use new default elevation 
- Updated relevant tests

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -43,9 +43,9 @@
     "webpack-merge": "^4.1.4"
   },
   "dependencies": {
-    "@material-ui/core": "^3.9.0",
+    "@material-ui/core": "^4.0.0-rc.0",
```
Comment (Joonpark13) [inline package.json]: should be `4.1.3` (latest at the time of writing), not rc

File: package.json [hunk 2]
```diff
@@ -43,9 +43,9 @@
...
     "@material-ui/icons": "^2.0.3",
-    "@material-ui/styles": "^3.0.0-alpha.10",
+    "@material-ui/styles": "^4.0.0-rc.0",
```
Comment (Joonpark13) [inline package.json]: Same with this, should be `4.1.2` (latest at time of writing)

**Comment by kevinjihwanlee** [comment]
[Material-UI 4 Migration](https://trello.com/c/DRoie45O/175-material-ui-4-migration)

**Review by Joonpark13 — CHANGES_REQUESTED** [review]
Looks like the tests are failing, and also I think you missed some instances of `<Typography />`.

File: package.json (hunk 1 above)
Comment (kevinjihwanlee) [reply package.json]: Done

File: package.json (hunk 2 above)
Comment (kevinjihwanlee) [reply package.json]: Done

**Comment by Joonpark13** [comment]
image

Seeing some clipping in the about page, probably has to do with the typography upgrade. Otherwise looks good.

**Review by Joonpark13 — APPROVED** [review]

**Pull request merged**